t/154-semaphore.t

```diff
@@ -118,3 +118,44 @@ hello, world
 --- no_shutdown_error_log
 semaphore gc wait queue is not empty
 --- SKIP
+
+
+
+=== TEST 3: exit before post_handler was called
+If gc is called before the ngx_http_lua_sema_handler and free the sema memory
```

Comment:
I can't reproduce this case with this test case, can you describe the reproduction conditions in detail?